plugins/football-scoreboard/logo_downloader.py

```
@@ -122,14 +121,14 @@ def download_missing_logo(sport_key: str, team_id: str, team_abbr: str, logo_pat
     except PermissionError as e:
         logger.error(f"Permission denied for {team_abbr}: {e}")
-        logger.error(f"Please run: sudo ./scripts/fix_perms/fix_assets_permissions.sh")
+        logger.error("Please run: sudo ./scripts/fix_perms/fix_assets_permissions.sh")
         return False
     except Exception as e:
         logger.error(f"Failed to download logo for {team_abbr}: {e}")
         # Try to create placeholder as fallback
         try:
             return create_placeholder_logo(team_abbr, logo_path)
-        except:
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)